src/plugins/intel_cpu/thirdparty/CMakeLists.txt

```diff
@@ -2,6 +2,12 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
+cmake_policy(SET CMP0069 NEW)
+set(CMAKE_POLICY_DEFAULT_CMP0069 NEW)
+
+cmake_policy(SET CMP0092 NEW)
+cmake_policy(SET CMP0091 NEW)
+cmake_policy(SET CMP0117 NEW)
```

Comment:
why do we need it here? If it's for MLAS specifically, can we move it to subfolder?

Reply:
Yes, it can be remove totally, this is a draft pr for testing, it will be removed in official PR.